book/src/queries/syntax.md

```diff
@@ -94,9 +94,25 @@ from employees
 select first_name
 ```
 
-## Parentheses
+## Expressions
 
-Parentheses — `()` — are used to give precedence to inner expressions.
+Many PRQL statements compute a value with an _expression_. Expressions follow
+the normal rules for computer languages. In the example below, note the use of
```

Comment:
```suggestion
PRQL is made up of  _expressions_, like `2 + 3` or `(1 + x) * y)`. In the example below, note the use of
```

WDYT?

Reply:
Yes, but the second expression needs an opening "("

Then LGTM